src/5e-SRD-StartingEquipment.json

```diff
@@ -6,60 +6,73 @@
-			"item": { "url": "/api/equipment/halberd", "name": "Halberd"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/lance", "name": "Lance"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/longsword", "name": "Longsword"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/maul", "name": "Maul"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/morningstar", "name": "Morningstar"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/pike", "name": "Pike"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/rapier", "name": "Rapier"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/scimitar", "name": "Scimitar"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/shortsword", "name": "Shortsword"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/trident", "name": "Trident"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/war-pick", "name": "War pick"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/warhammer", "name": "Warhammer"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/whip", "name": "Whip"}, "quantity": 1}]
-	}],
-	"choice_2": [{
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/handaxe", "name": "Handaxe"}, "quantity": 2}]
-	}, {
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/club", "name": "Club"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/dagger", "name": "Dagger"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/greatclub", "name": "Greatclub"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/handaxe", "name": "Handaxe"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/javelin", "name": "Javelin"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/light-hammer", "name": "Light hammer"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/mace", "name": "Mace"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/quarterstaff", "name": "Quarterstaff"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sickle", "name": "Sickle"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/spear", "name": "Spear"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/crossbow-light", "name": "Crossbow, light"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/dart", "name": "Dart"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/shortbow", "name": "Shortbow"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sling", "name": "Sling"}, "quantity": 1}]
-	}],
+		"item": { "url": "/api/equipment/javelin", "name": "Javelin"}, "quantity": 4}
+	],
+	"choices": [
+		{
+			"choose": 1,
+			"type": "choice",
+			"from": [
+				{
+					"choose": 1,
+					"type": "equipment",
+					"from": [{
+						"item": { "url": "/api/equipment/greataxe", "name": "Greataxe"}, "quantity": 1}]
+				}, {
+					"choose": 1,
+					"type": "equipment",
+					"from": [{
```

Comment:
The data is "one greataxe or any martial melee weapon". So we should have a choice of "one greataxe" and "one martial melee weapon":

```
{
  choose: 1
  type: equipment
  from: [
    {
      item: {
        url: "...",
        name: Greataxe }
      quantity
    },
    {
      choose: 1
      type: equipment
      from: {
        item_category: martial-melee-weapon
      }
    }
  ]
}
```

(this was roughly written, but the idea is there.

Reply:
Are we happy here to lose the `quantity` attribute for the nested options?

Reply:
The option says "choose 1 from", so that acts as quantity, right?